### PR TITLE
Update add-another-email-alias-for-a-user.md

### PR DESCRIPTION
**Pull request by msbemba**
Added admin role needed to create a alias.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6388

File: microsoft-365/admin/email/add-another-email-alias-for-a-user.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ You can create up to 400 aliases for a user. No additional fees or licenses are
   
 ## Add email aliases to a user
 
-You must have [admin permissions](../add-users/about-admin-roles.md) to do this. 
+You must have Global Admin rights to add email aliases to a user.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/email/add-another-email-alias-for-a-user.md:43]:
```suggestion
You must have Global Admin rights to add email aliases for a user.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Pull request merged**